app/src/main/java/com/anzelmas/quizzando/QuizQuestionActivity.kt

```diff
@@ -14,6 +15,145 @@ import androidx.databinding.DataBindingUtil
 import androidx.databinding.ViewDataBinding
 import kotlinx.android.synthetic.main.activity_quiz_question.*
 
+class QuizQuestionActivity : AppCompatActivity(), View.OnClickListener {
+
+
+    private var currentPosition: Int = 1
+    private var questionList: ArrayList<Question>? = null
+    private var selectedOption: Int = 0
+    private var correctAnswersNumber: Int = 0
+    private var userName: String? = null
+
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_quiz_question)
+
+        userName = intent.getStringExtra(Constants.USER_NAME)
+
+        questionList = Constants.getQuestions()
+//        Log.i("Question size", "${questionList!!.size}")
+
+        putQuestion()
+
+        optionOne_TextView.setOnClickListener(this)
+        optionTwo_textView.setOnClickListener(this)
+        optionThree_TextView.setOnClickListener(this)
+        optionFour_TextView.setOnClickListener(this)
+        submit_Button.setOnClickListener(this)
+    }
+
+    private fun putQuestion() {
+        val question = questionList!![currentPosition - 1]
+
+        defaultOptions()
+        submit_Button.text = "SUBMIT"
+
+        progressBar.progress = currentPosition
+        progressBar_textView.text = "$currentPosition" + "/" + progressBar.max
+
+
+        question_TextView.text = question.question
+        question_image_imageView.setImageResource(question.image)
+        optionOne_TextView.text = question.optionOne
+        optionTwo_textView.text = question.optionTwo
+        optionThree_TextView.text = question.optionThree
+        optionFour_TextView.text = question.optionFour
+    }
+
+    private fun defaultOptions() {
+        val options = ArrayList<TextView>()
+        options.add(0, optionOne_TextView)
+        options.add(1, optionTwo_textView)
+        options.add(2, optionThree_TextView)
+        options.add(3, optionFour_TextView)
+
+        for (option in options) {
+            option.setTextColor(Color.parseColor("#313030"))
```

Comment:
For colors - store color values in seperate xml file (e.g. `res/values/styles.xml`). https://stackoverflow.com/questions/6674183/storing-hex-color-values-in-strings-xml